assets/sass/modules/_enroll.scss

```diff
@@ -152,10 +152,16 @@ $enroll-content-spacing: 15px;
 
   .sms-request-button {
     float: right;
-    width: 111px;
+    width: 136px;
     margin-top: 0;
     padding: 0;
 
+    // This css will break the text into two lines if it's too long.
+    white-space: normal;
+    overflow: visible;
+    line-height: 1;
+    padding-top: 12px;
+
```

Comment:
Adjusted the width from line 140 and 155 here. Then instead of ellipsis on long text, I've made it to break onto the next line - 
![image](https://user-images.githubusercontent.com/62976351/103421668-6a6f1b80-4b52-11eb-8d06-5b90af0c6925.png)

With the current mark-up (anchor tag for button and no parent div around it), this is the best I could do for multi-line texts that wouldn't change vertical placement of text for single-line texts on that button. Please let me know if you have a better solution.

Reply:
Without these changes, it would have been - 
![image](https://user-images.githubusercontent.com/62976351/103421773-fbde8d80-4b52-11eb-84c2-13cf73fc9edb.png)

Note: `extra` part is something I added for testing purposes, it's not a part of the actual norwegian string.